docs/src/app/components/pages/components/BottomNavigation/ExampleSimple.js

```diff
@@ -0,0 +1,43 @@
+import React from 'react';
+import FontIcon from 'material-ui/FontIcon';
+import {BottomNavigation, BottomNavigationItem} from 'material-ui/BottomNavigation';
+import Paper from 'material-ui/Paper';
+
+const recentsIcon = <FontIcon className="material-icons">restore</FontIcon>;
+const favoritesIcon = <FontIcon className="material-icons">favorite</FontIcon>;
+const nearbyIcon = <FontIcon className="material-icons">location_on</FontIcon>;
+
+class BottomNavigationExampleSimple extends React.Component {
```

Comment:
I'll make these changes, although I don't think you should use non-standard syntax in your examples. It will probably confuse people who don't have it enabled in their compilers. It also seems to confuse React docgen when `render` is an arrow function.